dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -186,12 +192,17 @@ def main(ckpt_dir_path: str, ckpt_restore_path: str = ""):
         end = time.time()
 
         if torch.distributed.get_rank() == 0:
-            print(f"Loaded checkpoint from {new_ckpt_dir_path}.")
+            print(
+                f"Loaded checkpoint from {new_ckpt_dir_path} in {end - start} seconds"
+            )
         load_checkpoint_times.append(end - start)
 
     if torch.distributed.get_rank() == 0:
         avg_load_time = statistics.mean(load_checkpoint_times)
+        avg_save_time = statistics.mean(save_checkpoint_times)
         print_times(args, avg_save_time, avg_load_time)
+        print(f"All save times: {save_checkpoint_times}")
```

Comment:
nit: we could just put `if args.save_only:` around the print for loads to make output cleaner in that case.

Reply:
Done